testing/CMakeLists.txt

```diff
@@ -309,6 +344,7 @@ macro(add_unit_test test_name static_lib)
         ${ARGN})
     target_include_directories(${test_name} PRIVATE
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+        PRIVATE ${OPAE_SDK_SOURCE}/libopae/plugins/xfpga
```

Comment:
This makes xfpga include files visible to all tests.  While it may make sense for your tests in the short term, won't this dependency go away eventually?  Wondering if it makes more sense to only add this to the ASE tests?

Reply:
ASE doesn't need plugins/xfpga. I moved this include folder from the global include_directories() here since it caused ASE tests compilation errors.

Reply:
Wouldn't it make more sense to move this to the new `add_ase_test` macro then?